tsconfig.json

```diff
@@ -1,22 +1,26 @@
 {
-  "compilerOptions": {
-    "target": "es2017",
-    "module": "es2015",
-    "moduleResolution": "node",
-    "lib": ["esnext.array", "esnext", "es2017", "dom", "WebWorker"],
-    "declaration": true,
-    "declarationMap": true,
-    "sourceMap": true,
-    "inlineSources": true,
-    "strict": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true,
-    "noImplicitAny": true,
-    "noImplicitThis": true,
-    "experimentalDecorators": true
-  },
-  "include": ["src/**/*.ts", "types/*.ts"],
-  "exclude": ["node_modules", "dist"]
+  "extends": "./tsconfig.build.json"
 }
+
+// {
+//   "compilerOptions": {
+//     "target": "es2017",
+//     "module": "es2015",
+//     "moduleResolution": "node",
+//     "lib": ["esnext.array", "esnext", "es2017", "dom"],
+//     "declaration": true,
+//     "declarationMap": true,
+//     "sourceMap": true,
+//     "inlineSources": true,
+//     "strict": true,
+//     "noUnusedLocals": true,
+//     "noUnusedParameters": true,
+//     "noImplicitReturns": true,
+//     "noFallthroughCasesInSwitch": true,
+//     "noImplicitAny": true,
+//     "noImplicitThis": true,
+//     "experimentalDecorators": true
+//   },
+//   "include": ["src/**/*.ts", "types/*.ts"],
+//   "exclude": ["node_modules", "dist"]
+// }
```

Comment:
it is not valid to add // comments in a JSON file

Reply:
Done ✅